common/components/ui/SimpleButton.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, { Component } from 'react';
+
+const DEFAULT_BUTTON_TYPE = 'primary';
+const DEFAULT_BUTTON_SIZE = 'lg';
+
+const Spinner = () => {
+  return <i className="fa fa-spinner fa-spin fa-fw" />;
+};
```

Comment:
Would you mind if I made a PR on top of this to do this in CSS? We could kill the added FontAwesome dependency (Unless we've been itching for an icon font, which might not be bad to have.)